### PR TITLE
Add test/plan/apply action

### PR DESCRIPTION
**Pull request by mattwillsher**

### CODE REVIEW

File: .github/workflows/terraform.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: "Terraform"
+
+on:
+  push:
+    branches:
+      - master
```
Comment (mjdavies) [inline .github/workflows/terraform.yaml:6]: Is terraform racist as well and will accept master in place of main?

Comment (mattwillsher) [reply .github/workflows/terraform.yaml]: Fixed

**Comment by mattwillsher** [comment]
How useful this is I'm not really sure - we can either drive it from this side and the action or have TF Cloud pick it up... this way maybe gives more control?
Also gives option for other change (checkov, fmt included, can add docs and tf lint and such)

**Comment by mattwillsher** [comment]
Also upgrade to 0.13

**Review by treilly94 — APPROVED** [review]

**Pull request merged**